src/preferences/preferences_list.hpp

```diff
@@ -16,6 +16,8 @@
 
 #include "enum_base.hpp"
 
+#define ADDPREF(pref) static constexpr const char* const pref = #pref;
```

Comment:
Would this work as std::string_view?

Reply:
Hmm… it's a static string literal, so… maaaybe? I'm honestly not too sure.

Reply:
Is there any benefit to using a string_view?

Reply:
Generally good practice. Easier to pass to functions that take a string_view instead of `const char*`.

Reply:
Are there many functions that only take a string_view though?